cerberus-web/src/main/java/com/nike/cerberus/service/SafeDepositBoxService.java

```diff
@@ -309,6 +314,12 @@ public SafeDepositBoxV2 updateSafeDepositBoxV2(
     final Set<IamPrincipalPermission> iamRolePermissionSet =
         safeDepositBox.getIamPrincipalPermissions();
 
+    iamRolePermissionSet.stream()
```

Comment:
This section of code is identical to the code above starting on line 249, maybe it could be pulled into a method?

Reply:
Sure